components/salsa-2022/src/database.rs

```diff
@@ -11,7 +11,9 @@ pub trait Database: HasJarsDyn + AsSalsaDatabase {
         log::debug!("salsa_event: {:?}", event.debug(self));
     }
 
-    fn salsa_runtime(&self) -> &Runtime;
+    fn salsa_runtime(&self) -> &Runtime {
+        self.runtime()
+    }
```

Comment:
Give `salsa_runtime` a default implementation.  
I wanted to delete it directly, because we already have `db.runtime()`, but found that it is used in many places in the salsa code

Reply:
Those places can probably be rewritten to use `db.runtime()`?

Reply:
I think a default implementation is fine, though

Reply:
I removed `salsa_runtime` and replaced it with `db.runtime()` wherever it appears